### PR TITLE
Created the static pages for the about menu.

### PR DESCRIPTION
**Pull request by RohithTechdenali**
Created reusable components for paragraph, heading, strong text, and anchor tag.
Created the static pages for the About menu and added the About module.

image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by Vishnu9901 — APPROVED** [review]

File: src/routes/routes.tsx [hunk 1]
```diff
@@ -1,8 +1,15 @@
 
 import RegisterPage from '@pages/RegisterPage';
 import { AppSpinner } from '@ui/atoms/AppSpinner';
+import ContactUs from '@ui/organisms/About/ContactUs';
+import Culture from '@ui/organisms/About/Culture';
+import PeelLegacy from '@ui/organisms/About/OurPeelLegacy';
+import OurStory from '@ui/organisms/About/OurStory';
+import ReturnsAndExchange from '@ui/organisms/About/Returns&Exchange';
+import SubscriptionTerms from '@ui/organisms/About/Subscriptions';
```
Comment (Vishnu9901) [inline src/routes/routes.tsx:9]: Use lazy Loading to a above pages

File: src/routes/routes.tsx [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 import { lazy, Suspense } from 'react';
 import { Route, Routes } from 'react-router-dom';
+import AboutModule from 'src/modules/aboutModule/aboutModule';
```
Comment (Vishnu9901) [inline src/routes/routes.tsx:12]: use lazyloading

File: src/libs/ui/src/organisms/About/ContactUs.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import Anchor from '@ui/molecules/Anchor/Anchor';
```
Comment (Vishnu9901) [inline src/libs/ui/src/organisms/About/ContactUs.tsx:1]: wrap these organisms in pages and use those pages in routes page

Comment (Vishnu9901) [inline src/libs/ui/src/organisms/About/ContactUs.tsx:1]: Maintain Folder structure

**Review by Vishnu9901 — CHANGES_REQUESTED** [review]

**Comment by RohithTechdenali** [comment]
Made the changes as requested.

image
image
image

**Comment by Vishnu9901** [comment]
Hi @RohithTechdenali , Please check all page's Ui alignments, and, font styles's and padding margin should match with pcapro website. Add routing to pages from the header

- [x] Add routing to pages from the header

- [x]   Ui alignments, and, font styles's and padding margin should match with pcapro website

File: src/pages/ContactUsPage/index.tsx
Comment (Nishant8209) [inline src/pages/ContactUsPage/index.tsx:1]: Create one ContactUsTemplate in that template call the ContactUs component and call that template in the ContactUsPage

**Review by Vishnu9901 — APPROVED** [review]

**Pull request merged**